internal/pkg/featureflags/feature_flags.go

```diff
@@ -59,42 +61,44 @@ type launchDarklyManager struct {
 }
 
 func Init(cfg *v1.Config) {
-	platformName := cfg.Context().GetPlatform().GetName()
-	cliBasePath := fmt.Sprintf(baseURL, auth.CCloudURL, cliProdEnvClientId)
+	cliBasePath := fmt.Sprintf(baseURL, ccloudUrl, cliProdEnvClientId)
+	ccloudBasePath := fmt.Sprintf(baseURL, ccloudUrl, ccloudProdEnvClientId)
 	if cfg.IsTest {
 		cliBasePath = fmt.Sprintf(baseURL, testserver.TestCloudUrl.String(), "1234")
-	} else if platformName != cliProdEnv {
-		cliBasePath = fmt.Sprintf(baseURL, auth.CCloudURL, cliTestEnvClientId)
-	}
-
-	ccloudClientProvider := func(client v1.LaunchDarklyClient) *sling.Sling {
-		if cfg.IsTest || platformName != cliProdEnv {
-			return sling.New().Base(fmt.Sprintf(baseURL, auth.CCloudURL, ccloudCpdEnvClientId))
-		}
-
-		var ccloudBasePath string
-		switch client {
-		case v1.CcloudDevelLaunchDarklyClient:
-			ccloudBasePath = fmt.Sprintf(baseURL, auth.CCloudURL, ccloudDevelEnvClientId)
-		case v1.CcloudStagLaunchDarklyClient:
-			ccloudBasePath = fmt.Sprintf(baseURL, auth.CCloudURL, ccloudStagEnvClientId)
-		default:
-			ccloudBasePath = fmt.Sprintf(baseURL, auth.CCloudURL, ccloudProdEnvClientId)
+		ccloudBasePath = cliBasePath
+	} else {
+		switch cfg.Context().GetPlatform().GetName() {
+		case devel:
+			ccloudBasePath = fmt.Sprintf(baseURL, ccloudUrl, ccloudDevelEnvClientId)
+			cliBasePath = fmt.Sprintf(baseURL, ccloudUrl, cliTestEnvClientId)
+		case stag:
+			ccloudBasePath = fmt.Sprintf(baseURL, ccloudUrl, ccloudStagEnvClientId)
+			cliBasePath = fmt.Sprintf(baseURL, ccloudUrl, cliTestEnvClientId)
```

Comment:
There's a better way to write this that doesn't involve duplicate code

Reply:
Wouldn't we always want to default to `prod`